templates/entries.html

```diff
@@ -1,7 +1,32 @@
 <!DOCTYPE html>
 <html>
     <head>
-
+        <style>
+            /* Set height of the grid so .sidenav can be 100% (adjust if needed) */
+            .row.content {height: 1500px}
```

Comment:
Just changed the layout of the entry page, so that later I could pull in the profile picture without a prblem